console/src/components/Services/Data/RawSQL/RawSQL.js

```diff
@@ -96,9 +117,6 @@ const RawSQL = ({
   );
 
   const submitSQL = () => {
```

Comment:
We need to bring this logic back. The last ran SQL query should be stored in local storage. Users can for example close or refresh the page and current SQL won't be saved after running the query.

Reply:
@beerose This functionality is already taken care [here](https://github.com/soorajshankar/graphql-engine/blob/12a1968908b733ce188191aea60773d88cc8d7bf/console/src/components/Services/Data/RawSQL/RawSQL.js#L86) with `useEffect` hook.
I removed only the extra implementation.

Reply:
This won't work on page refresh. 
![2020-05-27 at 11 45 08 - Beige Frog](https://user-images.githubusercontent.com/9019397/83004205-a61b9300-a00f-11ea-9f10-5a5aace1904b.gif)

Reply:
Ohh okay got it!

Reply:
`useEffect` would only set this on unmounting the component.

Reply:
one more point, if the user has not clicked `Run!` and if he refreshes the page the data will be lost. 
I hope that should be okay.

Reply:
I am not sure if I understand correctly, but it should not be necessary for a user to run a query for it to be persisted. Any content I have in my editor should be persisted so that I can come back and continue.

Reply:
The current logic in master persists SQL in two cases:
1. The user clicks `run`.
2. The user leaves the component. 

It doesn't persist SQL if the user refreshes the page while being on Raw SQL page.

Reply:
This PR would leave this part of functionality as it was.